distribution/docs/src/main/resources/content/_configuring/reusing-configurations.adoc

```diff
@@ -23,7 +23,11 @@ To import previously exported configuration settings:
 . Launch the newly installed ${branding}.
 . Make sure to install and re-enable the ${branding} <<_starting_as_a_service_with_automatic_start_on_system_boot,service>> on the new system if it was installed and enabled on the original system
 . Copy the previously exported files from your secure locations to the `exported` directory underneath `${home_directory}`.
-. Run the command migration:import from the ${command-console}.
+. Either:
+.. Step through the installation process.
+.. Run the command migration:import from the ${command-console}.
+. Or if you want to restore the original profile along with the configuration (experimental):
```

Comment:
not sure if we use `you` by convention in the documentation, but an alternative might be `Or if an administrator wishes to restore the original profile...`